confidential-assets/src/confidentialAsset.ts

```diff
@@ -92,12 +92,13 @@ export class ConfidentialAsset {
     args: {
       tokenAddress: string;
       publicKey: HexInput | TwistedEd25519PublicKey;
-    } & Omit<InputGenerateSingleSignerRawTransactionArgs, "payload" | "aptosConfig">,
+      withFeePayer?: boolean;
+    } & Omit<InputGenerateSingleSignerRawTransactionArgs, "payload" | "aptosConfig" | "feePayerAddress">,
```

Comment:
nit: I would just define `options?: InputGenerateTransactionOptions, sender: AccountAddressInput` as args instead of the `Omit` =)

https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/transactions/types.ts#L322

Reply:
Oops automerged, I'll think about this on the next PR.